manifests/00-custom-resource-definition.yaml

```diff
@@ -131,10 +132,11 @@ spec:
                 description: "endpointPublishingStrategy is used to publish the ingress
                   controller endpoints to other networks, enable load balancer integrations,
                   etc. \n If unset, the default is based on infrastructure.config.openshift.io/cluster
-                  .status.platform: \n   AWS:      LoadBalancerService (with External
-                  scope)   Azure:    LoadBalancerService (with External scope)   GCP:
-                  \     LoadBalancerService (with External scope)   IBMCloud: LoadBalancerService
-                  (with External scope)   Libvirt:  HostNetwork \n Any other platform
+                  .status.platform: \n   AWS:          LoadBalancerService (with External
+                  scope)   Azure:        LoadBalancerService (with External scope)
+                  \  GCP:          LoadBalancerService (with External scope)   IBMCloud:
+                  \    LoadBalancerService (with External scope)   AlibabaCloud: LoadBalancerService
+                  (with External scope)   Libvirt:      HostNetwork \n Any other platform
```

Comment:
should PowerVS also be listed here?

Reply:
The PowerVS cannot be added to this file unless its added to api [here](https://github.com/Karthik-K-N/cluster-ingress-operator/blob/15576bd84b1a4474ac2cbdb2d277594405022270/vendor/github.com/openshift/api/operator/v1/0000_50_ingress-operator_00-ingresscontroller.crd.yaml#L131). The CI job `ci/prow/verify ` ([link](https://github.com/Karthik-K-N/cluster-ingress-operator/blob/630b4970512dbce40b5becd1063a99ba18f888d2/hack/verify-generated-crd.sh#L20-L19)) expects the contents to be identical.